front/src/components/student-registration/student-registration-form/student-registration-form.js

```diff
@@ -0,0 +1,203 @@
+import React, { Component } from 'react';
+import { studentRegistered } from '../../../common/redux/actions';
+import { Row, Form, Typography } from 'antd';
+import { Col, Input, Button } from '../student-registration-form/style';
+
+const { Title } = Typography;
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+
+const tailLayout = {
+  wrapperCol: {
+    offset: 9,
+    span: 16,
+  },
+};
+
+class StudentRegistrationPage extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.validateEng = this.validateEng.bind(this);
+    this.validateUkr = this.validateUkr.bind(this);
+    this.validatePhoneNumber = this.validatePhoneNumber.bind(this);
+    this.confirmPassword = this.confirmPassword.bind(this);
+  }
+
+  handleSubmit(data) {
+    data.groupToken = this.props.token;
+
+    this.props.dispatch(studentRegistered(data));
```

Comment:
Don't call `dispatch` here, it is better to define `mapDispatchToProps` object as a second argument to `connect` function